qiita_db/handlers/archive.py

```diff
@@ -0,0 +1,28 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+from .oauth2 import OauthBaseHandler, authenticate_oauth
+
+
+class APIArchiveObservations(OauthBaseHandler):
+    @authenticate_oauth
+    def post(self):
```

Comment:
Since this is retrieving information this should be a "get"

Reply:
After offline discussion it should be a POST. The requests can get large and there is a size limitation on the URL sent. Also, although it is possible to send the extra parameters on the body of the GET request, it is considered bad practice.